src/main/java/com/google/step/snippet/external/StackOverflowClient.java

```diff
@@ -91,6 +84,9 @@ private String getQuestionId(String url) {
     }
     // Parse the URL to get the question id.
     String[] segments = uri.getPath().split("/");
+    if (segments.length <= ID_INDEX || !segments[QUESTION_PATH_INDEX].equals(QUESTION_ID_INDEX)) {
```

Comment:
Ah sorry, for the confusion, I was thinking replacing `ID_INDEX` with `QUESTION_ID_INDEX` and keeping `QUESTION_PARAMETER` so:

`if (segments.length <= QUESTION_ID_INDEX || !segments[QUESTION_PATH_INDEX].equals(QUESTION_PARAMETER))`